### PR TITLE
Fix file migration generation

### PR DESCRIPTION
**Pull request by magicalbanana**
:man: :bear: :pig: 

Fix file generation for non public schemas.

### CODE REVIEW

**Comment by calebthompson** [comment]
Can you remove the Docker stuff, please?

File: lib/generators/scenic/view/view_generator.rb [hunk 1]
```diff
@@ -28,12 +28,12 @@ def create_migration_file
...
           migration_template(
             "db/migrate/create_view.erb",
-            "db/migrate/create_#{plural_file_name}.rb",
+            "db/migrate/create_#{file_name.gsub('.', "_").pluralize}.rb",
           )
         else
           migration_template(
             "db/migrate/update_view.erb",
-            "db/migrate/update_#{plural_file_name}_to_version_#{version}.rb",
+            "db/migrate/update_#{file_name.gsub('.', "_").pluralize}_to_version_#{version}.rb",
```
[the remark this answers is not in the document]
Comment (calebthompson) [reply lib/generators/scenic/view/view_generator.rb]:
```suggestion
            "db/migrate/update_#{file_name.gsub('.', '_').pluralize}_to_version_#{version}.rb",
```

File: lib/generators/scenic/view/view_generator.rb [hunk 2]
```diff
@@ -28,12 +28,12 @@ def create_migration_file
         if creating_new_view? || destroying_initial_view?
           migration_template(
             "db/migrate/create_view.erb",
-            "db/migrate/create_#{plural_file_name}.rb",
+            "db/migrate/create_#{file_name.gsub('.', "_").pluralize}.rb",
```
[the remark this answers is not in the document]
Comment (calebthompson) [reply lib/generators/scenic/view/view_generator.rb]:
```suggestion
            "db/migrate/create_#{file_name.gsub('.', '_').pluralize}.rb",
```

File: spec/dummy/config/database.yml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 development: &default
   adapter: postgresql
+  host: postgres
+  port: 5432
+  username: postgres
+  password: postgres
```
Comment (calebthompson) [inline spec/dummy/config/database.yml]: These aren't needed.

**Comment by calebthompson** [comment]
We also need a spec or specs showing what's changing here.

**Comment by derekprior** [comment]
👋 Thanks for the contribution. I tried to clean this up to merge it, but only after getting all the way through it did I realize I have no idea what it is that is broken that needs to be fixed. If you'd like to get this merged @magicalbanana, can you take another crack at a PR that:

1. does not add docker or change database.yml
2. has a more descriptive commit message
3. adds or modifies a test to exercise this behavior?

Thanks.

**Pull request closed without merging**